scripts/rnnlm/train_rnnlm.sh

```diff
@@ -25,7 +25,10 @@ embedding_lrate_factor=0.1  # the embedding learning rate is the
                             # nnet learning rate times this factor.
 backstitch_training_scale=0.0    # backstitch training scale
 backstitch_training_interval=1   # backstitch training interval
+
 cmd=run.pl  # you might want to set this to queue.pl
+queue_gpu_opt="--gpu 1" # you may change the GPU opt externally,
```

Comment:
@vesis84, please don't do this-- this is not really consistent with our normal way of working.
These kinds of things are configurable by changing queue.conf.

Reply:
Yes, originally I did this with 'queue.conf'. But is it possible somehow to have 2 queues, one with all the GPUs and other with GPUs that have more than, say 7G RAM? (the big GPUs are not always necessary...)